diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 76516184b49..d991ead6ff3 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -209,6 +209,9 @@ package_glibc() {
 
   # Provided by libxcrypt; keep the old shared library for backwards compatibility
   rm -f "$pkgdir"/usr/include/crypt.h "$pkgdir"/usr/lib/libcrypt.{a,so}
+
+  # Make copies of documentations
+  find "$srcdir/glibc/" -maxdepth 1 -type f | grep '/[A-Z.]\{1,\}$' | xargs install -Dm644 -t "$pkgdir/usr/share/docs/glibc"
 }
 
 package_lib32-glibc() {
@@ -246,4 +249,7 @@ package_lib32-glibc() {
 
   # Provided by lib32-libxcrypt; keep the old shared library for backwards compatibility
   rm -f "$pkgdir"/usr/lib32/libcrypt.{a,so}
+
+  # Make copies of documentations
+  find "$srcdir/glibc/" -maxdepth 1 -type f | grep '/[A-Z.]\{1,\}$' | xargs install -Dm644 -t "$pkgdir/usr/share/docs/lib32-glibc"
 }
