diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 22342a6..5004741 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -26,6 +26,7 @@ source=(https://ftp.gnu.org/gnu/glibc/glibc-$pkgver.tar.xz{,.sig}
         0001-localedef-Update-LC_MONETARY-handling-Bug-28845.patch
         0001-localedata-Do-not-generate-output-if-warnings-were-p.patch
         0001-elf-Fix-DFS-sorting-algorithm-for-LD_TRACE_LOADED_OB.patch
+        force-uname.patch
 )
 validpgpkeys=(7273542B39962DF7B299931416792B4EA25340F8 # Carlos O'Donell
               BC7C7372637EC10C57D7AA6579C43DFBF1CF2187) # Siddhesh Poyarekar
@@ -39,7 +40,8 @@ b2sums=('623c728884f070cd87ffeb9203f74206197c52405ac9bc44f3dd519a3468b8e8ae2536c
         'edef5f724f68ea95c6b0127bd13a10245f548afc381b2d0a6d1d06ee9f87b7dd89c6becd35d5ae722bf838594eb870a747f67f07f46e7d63f8c8d1a43cce4a52'
         '3c4de02d6308a4d39693e6effa08894ae096397347e439b93b0b5328aba41a373f7f2eeb88c109970f69cca65aaa74ba14e384d6d033f6a913438f4f920854a6'
         'b6014af23688ea971d7af38d58829c8016a0af63770b7f5e8bc986c13ea2cafefe0935ff483b1b37c87ebe3e652b6e0c48ab43dd8ae752d0ed7fecc3751432f4'
-        'd77c8f9c807f25aeeafa302858116f4adf903f46fe1db2035dbe998105d7206bbc6d3055119831ff7744eb721b05795866c80d99671c3a8f33d5386d80978248')
+        'd77c8f9c807f25aeeafa302858116f4adf903f46fe1db2035dbe998105d7206bbc6d3055119831ff7744eb721b05795866c80d99671c3a8f33d5386d80978248'
+        '5b3f6dbfc8923d2b89ddf6c99d79edc4b6785dadcd69c82a5236ef60d91589c981b786d279f74dcd49ba52944e3b0508f795b4975e16d22d11e03fdc8f42434b')
 
 prepare() {
   mkdir -p glibc-build lib32-glibc-build
@@ -61,6 +63,9 @@ prepare() {
   # Fix Sefault introduced by new DFS algorithm
   # https://bugs.archlinux.org/task/73938
   patch -Np1 -i "${srcdir}"/0001-elf-Fix-DFS-sorting-algorithm-for-LD_TRACE_LOADED_OB.patch
+
+  # Use uname, instead of DSO, to check kernel version
+  patch -p1 -i "$srcdir"/force-uname.patch
 }
 
 build() {
