diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index aa64c04..973dcc4 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -186,6 +186,10 @@ package_glibc() {
   # libraries too. Useful for gdb's catch command.
   install -Dm644 "$srcdir/sdt.h" "$pkgdir/usr/include/sys/sdt.h"
   install -Dm644 "$srcdir/sdt-config.h" "$pkgdir/usr/include/sys/sdt-config.h"
+
+  # Make copies of documentations
+  find "$srcdir/glibc/" -maxdepth 1 -type f | grep '/[A-Z.]\{1,\}$' | xargs install -Dm644 -t "$pkgdir/usr/share/doc/glibc"
+  install -Dm644 -t "$pkgdir/usr/share/licenses/glibc" "$srcdir/glibc/COPYING" "$srcdir/glibc/COPYING.LIB" "$srcdir/glibc/LICENSES"
 }
 
 package_lib32-glibc() {
@@ -210,4 +214,8 @@ package_lib32-glibc() {
 
   # Symlink /usr/lib32/locale to /usr/lib/locale
   ln -s ../lib/locale "$pkgdir/usr/lib32/locale"
+
+  # Make copies of documentations
+  find "$srcdir/glibc/" -maxdepth 1 -type f | grep '/[A-Z.]\{1,\}$' | xargs install -Dm644 -t "$pkgdir/usr/share/doc/lib32-glibc"
+  install -Dm644 -t "$pkgdir/usr/share/licenses/lib32-glibc" "$srcdir/glibc/COPYING" "$srcdir/glibc/COPYING.LIB" "$srcdir/glibc/LICENSES"
 }
