diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 4f77649..a6fb1c7 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -210,6 +210,10 @@ package_glibc() {
   # libraries too. Useful for gdb's catch command.
   install -Dm644 "$srcdir/sdt.h" "$pkgdir/usr/include/sys/sdt.h"
   install -Dm644 "$srcdir/sdt-config.h" "$pkgdir/usr/include/sys/sdt-config.h"
+
+  # Make copies of documentations
+  find "$srcdir/glibc/" -maxdepth 1 -type f | grep '/[A-Z.]\{1,\}$' | xargs install -Dm644 -t "$pkgdir/usr/share/doc/glibc"
+  install -Dm644 -t "$pkgdir/usr/share/licenses/glibc" "$srcdir/glibc/COPYING" "$srcdir/glibc/COPYING.LIB" "$srcdir/glibc/LICENSES"
 }
 
 package_lib32-glibc() {
@@ -244,4 +248,8 @@ package_lib32-glibc() {
       -not -name 'libthread_db.so*' \
       -name '*.so*' -type f -exec strip $STRIP_SHARED {} + 2> /dev/null || true
   fi
+
+  # Make copies of documentations
+  find "$srcdir/glibc/" -maxdepth 1 -type f | grep '/[A-Z.]\{1,\}$' | xargs install -Dm644 -t "$pkgdir/usr/share/doc/lib32-glibc"
+  install -Dm644 -t "$pkgdir/usr/share/licenses/lib32-glibc" "$srcdir/glibc/COPYING" "$srcdir/glibc/COPYING.LIB" "$srcdir/glibc/LICENSES"
 }
